_data/models.json

```diff
@@ -0,0 +1,122 @@
+[
+    {
+        "name": "GPT",
+        "id": "gpt",
+        "company": "openai",
+        "urls": [
+            "https://platform.openai.com/docs/models",
+            "https://huggingface.co/openai"
+        ],
+        "apis": [
+            "OpenAI"
+        ],
+        "glossary": true,
+        "formality": true,
+        "adaptive": true,
+        "fine_tuning": true,
+        "self_serve": true
+    },
+    {
+        "name": "Sonnet",
+        "id": "sonnet",
+        "company": "anthropic",
+        "urls": [
+            "https://docs.anthropic.com/en/docs/about-claude/models/overview"
+        ],
+        "apis": [
+            "Claude AI"
+        ]
+    },
+    {
+        "name": "DeepSeek AI",
+        "id": "deepseek-ai",
+        "company": "deepseek",
+        "urls": [
+            "https://huggingface.co/deepseek-ai"
+        ],
+        "apis": [
+            "DeepSeek"
+        ],
+        "fine_tuning": true,
+        "self_serve": true
+    },
+    {
+        "name": "TowerLLM",
+        "id": "towerllm",
+        "company": "unbabel",
+        "urls": [
+            "https://huggingface.co/collections/Unbabel/tower-plus-6846ca452a10c0905dc03c0f",
+            "https://unbabel.com/meet-towerllm"
+        ],
+        "apis": [
+            "Widn"
+        ],
+        "fine_tuning": true,
+        "self_serve": true
+    },
+    {
+        "name": "Sarvam",
```

Comment:
seems like its name is Sarvam-Translate

so maybe we should make the name and id reflect that?